internal/incoming/odfi/processor.go

```diff
@@ -180,6 +180,15 @@ func processFile(path string, auditSaver *AuditSaver, fileProcessors Processors)
 	return nil
 }
 
+func populateHashes(file *ach.File) {
+	for i := range file.Batches {
+		entries := file.Batches[i].GetEntries()
+		for j := range entries {
+			entries[i].ID = hash([]byte(entries[j].String()))
```

Comment:
Could it be the case that the two entries are precisely the same? Should this be a UUID instead?

As a consumer of the `ach` and `achgateway` it would feel weird that we deal with UUID in some parts, but hash in other parts.

No a big deal, but worth to document at the very least so people don't make the wrong assumptions.

Reply:
TraceNumbers within a file/batch can't be the same, so the hash shouldn't ever be the same under Nacha compliant conditions. 

Where are UUID's created in ach/achgateway?

Reply:
In the `ach` itself, it seems to put IDs into the information

Reply:
Correct so I'm not sure where UUID's come from.